rules-tests/CodeQuality/Rector/MethodCall/AssertComparisonToSpecificMethodRector/Fixture/fixture.php.inc

```diff
@@ -26,8 +26,8 @@ final class MyEqualsTest extends \PHPUnit\Framework\TestCase
 {
     public function test()
     {
-        $this->assertGreaterThan(2, count($something));
-        $this->assertGreaterThan($something[1]['foo'], $something[0]['foo']);
+        $this->assertLessThan(2, count($something));
+        $this->assertLessThan($something[1]['foo'], $something[0]['foo']);
```

Comment:
this seems invalid

```
        $something = [
            0 => [
                'foo' => 2,
            ],
            1 => [
                'foo' => 1,
            ],
            'bar',
        ];

        $this->assertTrue(count($something) > 2);
        $this->assertTrue($something[0]['foo'] > $something[1]['foo']);

        $this->assertLessThan(2, count($something));
        $this->assertLessThan($something[1]['foo'], $something[0]['foo']);
```

got:

```
There was 1 failure:

1) ArrayLookup\Tests\FilterTest::test
Failed asserting that 3 is less than 2.

/Users/samsonasik/www/GithubModules/ArrayLookup/tests/FilterTest.php:168
```

Reply:
I got it, when on `Greater`, `GreaterOrEqual`, `Smaller`, `SmallerOrEqual`, order of argument should be kept to avoid invalid comparison https://github.com/rectorphp/rector-phpunit/pull/532/commits/58115fd05d5687d13f18adc8209e793db6ce430f